### PR TITLE
DTSSTCI 44 - Add sptribs case api and frontend

### PR DESCRIPTION
**Pull request by gmmagruder**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DTSSTCI-44

### Change description ###
Added new ST group and sptribs-case-api amd sptribs-frontend to microservices.json. Also added st.html.

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: docs/c4/hmcts.mdsl [hunk 1]
```diff
@@ -160,6 +160,10 @@ pre = softwareSystem "PRE" {
    pre_api = container "PRE API" "Backend for Pre-Recorded Evidence project"
    pre_portal = container "PRE Portal" "Frontend for Pre-Recorded Evidence project"
 }
+st = softwareSystem "ST" {
+   sptribs_frontend = container "Special Tribunals Frontend" ""
```
Comment (tomxelliott) [inline docs/c4/hmcts.mdsl:164]: Can you add a description of the app in the second set of speech marks

File: docs/c4/hmcts.mdsl [hunk 2]
```diff
@@ -160,6 +160,10 @@ pre = softwareSystem "PRE" {
...
+   sptribs_case_api = container "Special Tribunal Case API" ""
```
Comment (tomxelliott) [inline docs/c4/hmcts.mdsl:165]: Can you add a description of the app in the second set of speech marks

File: docs/microservices.json [hunk 3]
```diff
@@ -3029,6 +3033,72 @@
...
       "apis": [],
       "version": null
-  }
+  },
+    {
+      "id": "sptribs-frontend",
+      "name": "Special Tribunals Frontend",
+      "group": "ST",
+      "type": "web",
+      "description": "",
```
Comment (tomxelliott) [inline docs/microservices.json:3042]: Can we add a short description please

File: docs/microservices.json [hunk 4]
```diff
@@ -3029,6 +3033,72 @@
...
+      ],
+      "apis": [],
+      "version": null
+    },
+    {
+      "id": "sptribs-case-api",
+      "name": "Special Tribunal Case API",
+      "group": "ST",
+      "type": "api",
+      "description": "",
```
Comment (tomxelliott) [inline docs/microservices.json:3075]: Can we add a short description please

**Review by tomxelliott — CHANGES_REQUESTED** [review]
Just need some descriptions of the apps

File: docs/microservices.json [hunk 5]
```diff
@@ -3029,6 +3033,72 @@
...
+      ],
+      "apis": [],
+      "version": null
+    },
+    {
+      "id": "sptribs-case-api",
+      "name": "Special Tribunal Case API",
+      "group": "ST",
+      "type": "api",
+      "description": "Java service to deal with special tribunal case management system",
```
Comment (tomxelliott) [inline docs/microservices.json:3075]:
```suggestion
      "description": "Business logic for the management of Special Tribunals cases",
```

File: docs/c4/hmcts.mdsl [hunk 6]
```diff
@@ -160,6 +160,10 @@ pre = softwareSystem "PRE" {
    pre_api = container "PRE API" "Backend for Pre-Recorded Evidence project"
    pre_portal = container "PRE Portal" "Frontend for Pre-Recorded Evidence project"
 }
+st = softwareSystem "ST" {
+   sptribs_frontend = container "Special Tribunals Frontend" "Special Tribunals Frontend"
+   sptribs_case_api = container "Special Tribunal Case API" "Java service to deal with special tribunal case management system"
```
Comment (tomxelliott) [inline docs/c4/hmcts.mdsl:165]:
```suggestion
   sptribs_case_api = container "Special Tribunal Case API" "Business logic for the management of Special Tribunals cases"
```

File: docs/microservices.json (hunk 5 above)
Comment (gmmagruder) [reply docs/microservices.json]: Done

**Review by tomxelliott — APPROVED** [review]

**Pull request merged**